src/foam/nanos/dig/DIG.js

```diff
@@ -11,14 +11,19 @@ foam.CLASS({
 
   documentation: 'Data Integration Gateway - Perform DAO operations against a web service',
 
-  requires: ['foam.net.web.HTTPRequest'],
+  requires: [
+    'foam.net.web.HTTPRequest'
+  ],
+
+  implements: [
+    'foam.mlang.Expressions',
+  ],
 
   tableColumns: [
     'id',
     'daoKey',
     'cmd',
-    'format',
-    'owner'
```

Comment:
Why removing owner?

Reply:
owner isn't set or used. not sure who put it in there but it should actually be createdBy.